lib/filewatch/boot_setup.rb

```diff
@@ -0,0 +1,91 @@
+require "rbconfig"
+
+module FileWatch
```

Comment:
point taken. I think, when there were few classes, it was fine to have it as it was before; however there was some duplication of repeated predicate `in windows?` tests when in tight loops etc. Plus I wanted all the requisites loaded in one place instead of requiring them again and again. It seemed to make sense to me to collect all the constants here too.
:+1: to documenting stuff.